### PR TITLE
A small batch of fixes and sprite improvements

### PR DESCRIPTION
**Pull request by ThePainkiller**
## About The Pull Request

- Fixes Pancakes, Eggs Benedict, Slime core soup and boiled slime core not having a proper icon state set
- Adds new trash items for waffles and icecreams
- New sprites for the Plant Analyzer and better sprites for robot components, each tier of robot armor now has their own unique sprite as well!
- Better sprites for Waffles (and their kind), Fish n' chips and THE cookie!
- Added Dumplings, Bacon Burger, BLT, Chicken Burger, Muffin Egg Sandwich, Tonkatsu, Bear Meat Chili, Beef Curry, Chicken Curry, Mashed Potatoes, Brownies (sliceable), three flavors of Icecream, Icecream Sandwiches, and Croissants.
- Reverts adding the extra lightbulbs to the swamp house on Kaz's idea, leaves the bed for doing surgery though.

Yes I know it's conflicting but I'll solve conflicts after pushing and rebasing with master, don't worry

<hr>

## Changelog
:cl:
add: Added even MORE food recipes! Details:

**Croissant:** 5u sugar + 5u milk + 1 raw dough slice + 1 butter slice
**Dumplings:** 1 cabbage + 5u soy sauce  + 1u salt + 1u pepper + 1u corn oil + 2 raw cutlets
**Muffing egg sandwich:** 1 bun + 1 fried egg + 1 bacon
**Bacon burger:** 1 bun + 1 patty + 1 bacon
**Chicken burger:** 1 bun + 1 fried poultry
**Tonkatsu:** 1u salt + 5u flour + 3u egg + 2u corn oil + 1 raw cutlet
**Brownies:** 10u Cocoa powder + 15u flour + 3u egg + 5u sugar + 1 whole stick of butter
**BLT:** 1 cabbage + 1 tomato + 2 bread slices + 1 bacon
**Sashimi:** 5u soy sauce + 2 carp fillets
**Beef curry:** 1 chili + 1 carrot + 1 tomato + 5u flour + 5u soy sauce + 1 raw meat + 1 butter slice + 1 boiled rice
**Chicken curry:** 1 chili + 1 carrot + 1 tomato + 5u flour + 5u soy sauce + 1 poultry breast + 1 butter slice + 1 boiled rice
**Mashed potatoes:** 5u milk + 2 potatoes + 1 butter slice
**Icecream sandwich:** Use a Cookie on a vanilla icecream

For icecreams, combine in a beaker:
**Vanilla icecream:** 5u cream + 5u milk + 5u sugar + 5u ice
**Chocolate Icecream:** 5u cream + 5u ice + 5u cocoa powder
**Strawberry Icecream:** 5u cream + 5u ice + 5u berryjuice

**RECIPE CHANGES:**
Katsudon now requires 1 Tonkatsu to craft instead of a Patty
Fish fingers are now crafted with: 5u flour + 3u egg + 2u corn oil + 1 carp meat
Tacos are now crafted with: 5u flour + 1 corn + 1 cabbage + 1 cutlet (cooked) + 1 cheese wedge
Birthday cakes and Brain cakes now also require a stick of butter besides their usual reageants

tweak: Reverts swamp house light bulbs added, leaves the bed intact. Adds borg limbs chance of drop to salvageable machines.
fix: Fixed certain food items not having properly assigned icon states
imageadd: New sprites for robot components, each tier of robot armor now has their own unique sprite as well! Better sprites for waffles, fish n' chips and the cookie, adds icons for Dumplings, Bacon Burger, BLT, Chicken Burger, Muffin Egg Sandwich, Tonkatsu, Bear Meat Chili, Beef Curry, Chicken Curry, Mashed Potatoes, Brownies, Icecream, Chocolate Icecream, Strawberry Icecream. Icecream Sandwiches and Croissants.
imagedel: goodbye dated waffle sprites
/:cl:

### CODE REVIEW

**Comment by ThePainkiller** [comment]
Oh, I take it the lights were already taken off then, I'm gonna revert the map change then

File: code/modules/reagents/recipes/recipes_food_drinks.dm [hunk 1]
```diff
@@ -156,6 +156,39 @@
 	for(var/i = 1, i <= created_volume, i++)
 		new /obj/item/reagent_containers/food/snacks/sliceable/butterstick(location)
 
+/datum/chemical_reaction/vanillaicecream
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/recipes/recipes_food_drinks.dm:159]: none of these conflict with milkshake?

Comment (ThePainkiller) [reply code/modules/reagents/recipes/recipes_food_drinks.dm:159]: God damn it. My next best bet was to add temperature requirements but it'd be impossible for a cook to have either instant ice or the means to procure it. How do you suggest we solve this recipe?

Comment (ThePainkiller) [reply code/modules/reagents/recipes/recipes_food_drinks.dm:166]: Alright, I've created a sub-reagent to work for icecream recipes, making Milkshake also be crafted by combining it with milk.
I'm also leaving this reagent for the future if someone wants to add more icecreams, they just have to combine it with their choice of flavor to make icecreams, this should solve all recipe conflicts

**Pull request merged**